spec/requests/entities_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+RSpec.describe '/entities', type: :request do
+  before(:each) do
+    @user = FactoryBot.create(:user)
+    @group = FactoryBot.create(:group, user: @user)
+    @entity = FactoryBot.create(:entity, user: @user, group: @group)
+
+    sign_in @user
+  end
+
+  describe 'GET /index' do
+    before(:each) do
+      get group_entities_path(@group)
+    end
+    it 'renders a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'body should includes correct placeholder text' do
+      expect(response.body).to include('Category Entities')
+      expect(response.body).to include @group.name
+      expect(response.body).to include @group.icon
+      expect(response.body).to include @entity.name
+    end
+  end
```

Comment:
hello @brytebee . Thanks for the feedback.  But I did not get  this error

This is my test results

![Screenshot 2022-11-22 at 20 21 06](https://user-images.githubusercontent.com/34389326/203413471-92c52ca0-9c79-40fb-83cc-b6eb4c214f78.png)


You need to run bundle install to have all dependencies setup. You are facing that issue because authentication helper is not working at you end

If I comment out the sign in helper like the coed below i will get the error you are facing


![Screenshot 2022-11-22 at 20 22 54](https://user-images.githubusercontent.com/34389326/203413794-31aa0552-7ba3-4adf-ad06-e904cf04e431.png)


You need to make sure that you set up everything correctly.

Reply:
Okay @learnwithalfred Kindly request another review if you've made the other changes and allow the next reviewer to check it. It may be an error on my end.